src/ontology/mondo.Makefile

```diff
@@ -264,7 +264,38 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
-	
+#############################################
+##### One-time scripts ######################
+#############################################
+# MedGen conflicts update
+## - main pipeline
+tmp/July2023_CUIReports_FromMedGentoMondo.xlsx:
+	mkdir -p tmp && wget "https://github.com/monarch-initiative/mondo-ingest/files/12029712/July2023_CUIReports_FromMedGentoMondo.xlsx" -O $@
+
+tmp/bad-medgen-xrefs.txt: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx
+	python ../scripts/bad_medgen_xrefs_collate.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -o $@
+
+# Creates a command that, when run, will create mondo-edit.obo.tmp files
+tmp/bad-medgen-xrefs-grep-command.sh tmp/bad-medgen-xrefs-grep-command-stricter.sh: tmp/bad-medgen-xrefs.txt
+	python ../scripts/bad_medgen_xrefs_grep_command.py -i tmp/bad-medgen-xrefs.txt -t mondo-edit.obo -o $@
+
+mondo-edit.obo.tmp: tmp/bad-medgen-xrefs-grep-command.sh
+	sh $<
+
+mondo-edit.obo.tmp-stricter: tmp/bad-medgen-xrefs-grep-command-stricter.sh
+	sh $<
+
+tmp/mondo-edit.obo.tmp.diff: mondo-edit.obo.tmp
+	diff mondo-edit.obo mondo-edit.obo.tmp > $@
+
+tmp/report-qc-medgen-conflicts-update-diff.tsv: tmp/mondo-edit.obo.tmp.diff
+	python ../scripts/bad_medgen_xrefs_update_analyze_diff.py -i tmp/mondo-edit.obo.tmp.diff -o $@
+
+# address-medgen-conflicts: https://github.com/monarch-initiative/mondo-ingest/issues/273
+.PHONY: address-medgen-conflicts
+address-medgen-conflicts: tmp/report-qc-medgen-conflicts-update-diff.tsv mondo-edit.obo.tmp-stricter
```

Comment:
Nico will handle these 6 cases:
```
non_xref_removals_content     "synonym: ""Ullrich disease"" RELATED [DOID:0050558, https://www.ncbi.nlm.nih.gov/gtr/conditions/C0410179/]
relationship: excluded_subClassOf MONDO:0016697 {source=""UMLS:CN201941""} ! low grade ependymoma
comment: Other ontologies classify peripheral neuroepithelioma as a seprate class but NCIT treats these as synonyms and we aim to follow NCIT here. (See HP:0006717, SNOMED:254764001, the ""included"" disease concept on OMIM:612219, and UMLS:C3489398).
synonym: ""polysyndactyly"" EXACT [SCTID:84598000, UMLS:C0265553]
synonym: ""thrombophilia, hereditary"" EXACT [MESH:C540694, UMLS:C2584620]
synonym: ""triple positive breast cancer"" EXACT [UMLS:C5243562]
```

Options:
a. Nico will make changes in a commit here (not preferred; I tend to force push and amend commits)
b. Nico will make changes in master and Joe will rebase (preferred)
c. Nico will make changes after this PR has been merged.

Reply:
Nico says don't worry about it. Global scale in mondo for this kind of thing; outdated refs. manually